kolibri/core/tasks/test/taskrunner/test_job_running.py

```diff
@@ -115,9 +114,10 @@ class EventProxy(object):
     creation.
     """
 
-    def __init__(self, *args, **kwargs):
-        self.event_id = uuid.uuid4().hex
-        EVENT_PROXY_MAPPINGS[self.event_id] = Event(*args, **kwargs)
+    def __init__(self, event_id=uuid.uuid4().hex, *args, **kwargs):
```

Comment:
Should set `event_id=None` here - as the default is evaluated in module scope, not at each invocation of the `__init__` function.

Can then, in the body of the `__init__` function do this:
```
if event_id is None:
    event_id = uuid.uuid4().hex
```